daprdocs/content/en/developing-applications/building-blocks/configuration/configuration-api-overview.md

```diff
@@ -0,0 +1,26 @@
+---
+type: docs
+title: "Configuration overview"
+linkTitle: "Overview"
+weight: 1000
+description: "Overview of the configuration API building block"
+---
+
+## Introduction
+
+Consuming application configuration is a common task when writing applications and frequently configuration stores are used to manage this configuration data. A configuration item is often dynamic in nature and is tightly coupled to the needs of the application that consumes it. For example, common uses for application configuration include names of secrets, different identifiers, partition or consumer IDs, names of databases to connect to etc. These configuration items are typically stored as key/value items in a state store or database.
+
+Dapr provides a [state management API]({{<ref "state-management-overview.md">}}) that is based on key-value stores to save and retrieve state. However, application configuration can be changed by either developers or operators at runtime and the developer needs to be notified of these changes in order to take the required action and load the new configuration. Also the configuration data may want to be read only. Dapr's Configuration API allows developers to consume configuration items that are returned as key/value pairs and subscribe to changes whenever a configuration item changes.
```

Comment:
Why mention `Dapr provides a [state management API]({{<ref "state-management-overview.md">}}) that is based on key-value stores to save and retrieve state.` here? If it's to clarify the difference between state management and configuration APIs, might be helpful to add a sentence about that explicitly (e.g. "Unlike the state management API, etc. etc.")

Reply:
You are correct. This is misleading adding the reference to state store here. I will remove this sentence, it does not add anything to the context and if anything makes it confusing to the reader.